plasmapy/diagnostics/tests/test_proton_radiography.py

```diff
@@ -43,6 +43,16 @@ def _test_grid(name, L=1 * u.mm, num=100, B0=10 * u.T):
 
     grid = CartesianGrid(-L, L, num=num)
 
+    defaults = {
+        "B0": 10 * u.T,
+        "E0": 5e8 * u.V / u.m,
+        "phi0": 1.4e5 * u.V,
+    }
+
+    for k in defaults.keys():
+        if k not in kwargs.keys():
+            kwargs[k] = defaults[k]
+
```

Comment:
Why not just add these to the default arguments? Won't that work the same way?

Reply:
Hm... yep you're right. 

I think I've been doing this sometimes when I have an unwieldly long list of possible arguments, but the way you suggest is definitely better here.